manifests/repo/download_postgresql_org.pp

```diff
@@ -17,14 +17,14 @@
     $label2 = 'rhel'
   }
 
-  yumrepo { 'yum.postgresql.org':
```

Comment:
This is going to leave behind old yum repo files, unless someone has a purge policy on yum repo:

    resources { "yumrepo": ensure => purged }

Umm, consider a) not renaming the resource - or b) explicitly removing the old resource, by making this old one ensure => absent? Anyone have any other ideas?